src/olympia/abuse/tests/test_views.py

```diff
@@ -166,6 +166,24 @@ def test_message_can_be_blank_if_reason_is_provided(self):
                           u'[Extension] Abuse Report for %s' % addon.name)
         assert report.message == ''
 
+    def test_message_length_limited(self):
+        addon = addon_factory()
+
+        response = self.client.post(
+            self.url,
+            data={'addon': str(addon.id),
+                  'message': 'a' * 10000})
+        assert response.status_code == 201
+
+        response = self.client.post(
+            self.url,
+            data={'addon': str(addon.id),
+                  'message': 'a' * 10001})
+        assert response.status_code == 400
+        assert json.loads(response.content) == {
+            'message': ['Ensure this field has no more than 10000 characters.']
```

Comment:
Could we configure a slightly better message? see: https://github.com/mozilla/addons-frontend/issues/8385#issuecomment-514990492

Reply:
"Please ensure this field has no more than 10000 characters"? I couldn't think of anything nicer.

Reply:
`Ensure this field has fewer than 10000 characters.` is less correct (because fewer than is `<10000` rather than `<=10000`) but is a little shorter?

Reply:
I think prepending the message with "Please" would make it more user friendly.